src/dispatching.rs

```diff
@@ -187,12 +187,36 @@
 //! useful features. See [`examples/dispatching_features.rs`] as a more involved
 //! example.
 //!
+//! ## Dispatching or REPLs?
+//!
+//! The difference between dispatching and the REPLs ([`crate::repl`] & co) is
+//! that dispatching allows you a greater degree of flexibility at the cost of a
+//! bit more complicated setup.
+//!
+//! Here are things that dispatching can do, but REPLs can't:
+//!  - Handle different kinds of [`Update`];
+//!  - [Pass dependencies](struct.DispatcherBuilder.html#method.dependencies) to
+//!    handlers;
+//!  - Disable a [default Ctrl-C
+//!    handling](struct.DispatcherBuilder.html#method.enable_ctrlc_handler);
+//!  - Control your
+//!    [default](struct.DispatcherBuilder.html#method.default_handler) and
+//!    [error](struct.DispatcherBuilder.html#method.error_handler) handlers;
+//!  - Use [dialogues](dialogue/index.html).
+//!  - Use [`dptree`]-related functionality.
+//!  - Probably more.
+//!
+//! Thus, REPLs are good for simple bots and rapid prototyping, but for more
+//! involved scenarios, we recommend using [`DispatcherBuilder`]/[`Dispatcher`]
```

Comment:
Is there a reason to link `DispatcherBuilder` here?

Reply:
`DispatcherBuilder` provides more methods than `Dispatcher`, so I think people will find it helpful.

Reply:
The only way to create `Dispatcher` is to go through `DispatcherBuilder` anyway though

Reply:
Maybe we should paraphrase to `, but for more involved scenarios, we recommend using dispatcher instead of REPLs.`?

Reply:
Given that this is the docs for the `dispatching` module this may be a good option, yes.

Reply:
Fixed.